app/pages/settings/email.jsx

```diff
@@ -308,13 +296,23 @@ class EmailSettingsPage extends React.Component {
             <Translate content="emailSettings.project.section" />
           </strong>
         </p>
+        <AutoSave resource={this.props.user}>
+          <label>
+            <input
+              type="checkbox"
+              name="project_email_communication"
+              checked={this.props.user.project_email_communication}
+              onChange={handleInputChange.bind(this.props.user)}
+            />{' '}
+            <Translate content="emailSettings.general.classify" />
+            <Translate component="p" content="emailSettings.general.note" />
+            <Translate component="p" content="emailSettings.general.manual" />
+          </label>
+        </AutoSave>
         <table>
           <thead>
             <tr>
-              <th>
-                <i className="fa fa-envelope-o fa-fw" />
-              </th>
-              <Translate component="th" content="emailSettings.project.header" />
+              <th />
```

Comment:
Table headings shouldn't be empty.

Reply:
OK. I couldn't find any advise on this. Why shouldn't they? I left it empty because leaving the mail icon didn't seem to make sense.

Reply:
They describe the content of the column to a screen reader. I'll post some links.

Reply:
https://webaim.org/techniques/tables/data

Reply:
https://teachaccess.github.io/tutorial/#/9

Reply:
Thank you!